packages/components/src/components/user/CreateUsersModal.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react'
```

Comment:
```suggestion
import React from 'react'
```

Reply:
Thanks. Looks like I've done this a bunch lately with the SM perm/user management stories. Fixing them all up now.